ui/src/app/pages/data/concept/concept-homepage.tsx

```diff
@@ -266,14 +266,19 @@ export const ConceptHomepage = withCurrentWorkspace()(
       }
     }
 
-    triggerSearch(e) {
+    handleChange(e) {
```

Comment:
Naming nit: since this is handling a change on a specific subcomponent of the concept homepage, I'd qualify the name a bit, e.g.

handleSearchChange

here and below (for handleSubmit)